File: Common/Entity/src/main/java/it/ey/dto/OVPStakeHolderDTO.java
package it.ey.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.*;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class OVPStakeHolderDTO {
    private Long id;
    @JsonIgnore
    private OVPDTO ovp;
    private StakeHolderDTO stakeholder;

}

File: dfp-piao-fe/src/app/shared/models/classes/ovp-stakeholder-dto.ts
import { StakeHolderDTO } from './stakeholder-dto';

export class OVPStakeHolderDTO {
  id?: number;
  stakeholder?: StakeHolderDTO;

  toString(): string {
    return this.stakeholder?.nomeStakeHolder + ' - ' + this.stakeholder?.relazionePA;
  }
}
